README.md

```diff
@@ -46,22 +45,11 @@ GOV.UK Publishing Components shares the same standards in terms of browser and a
 
 [See the rubydoc.info documentation](http://www.rubydoc.info/gems/govuk_publishing_components)
 
-## Working locally
+## Technical documentation
 
-### Install dependencies
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) or the local `startup.sh` script to run the app. Read the [guidance on local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) to find out more about each approach, before you get started.
 
-As well as the Ruby, you will also need the [Yarn package manager](https://classic.yarnpkg.com/en/docs/install/) to allow installing [npm](https://npmjs.com) dependencies.
-
-Install the application's dependencies with:
-
-```sh
-bundle install
-yarn install
```

Comment:
Has this bit been deleted entirely or moved somewhere else? Isn't it important?

Reply:
This is covered by GOV.UK Docker. Since a lot of frontend repos require both of these commands, we should list them in [the guidance for local frontend dev](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) if we think they're not obvious (when following the non-GOV.UK Docker path). Since they're not listed there right now, my assumption was that people either use GOV.UK Docker, or know enough about frontend dev to run them by themselves (but I might be wrong there).

Reply:
I see no reference of yarn in the mentioned documentation, hence there's a great value in having them here, especially for a new started who may be less familiar with it.